sdks/rust/src/sdk.rs

```diff
@@ -60,6 +60,35 @@ impl Sdk {
         )
         .parse()?;
 
+        Self::new_internal(addr, keep_alive).await
+    }
+
+    pub async fn new_with_host(
```

Comment:
🤔  should this be wrapped in `new` ?

Reply:
Since it has a default that is backward compatible, I think it's fine to have just the `new` method? Unless I am missing something in the details?